src/main/java/org/openjdk/jextract/impl/OutputFactory.java

```diff
@@ -222,7 +222,7 @@ private boolean generateFunctionalInterface(Type.Function func, String javaName)
 
         //generate functional interface
         if (func.varargs() && !func.argumentTypes().isEmpty()) {
-            warn("varargs in callbacks is not supported: " + func);
+            warn("varargs in callbacks is not supported: " + CDeclarationPrinter.declaration(func, javaName));
```

Comment:
This is a quality of life fix - on Windows, a very very big message is reported as we print out the function descriptor of the corresponding C function. By turning that into a C type (using same logic we use for javadoc generation) the warning messages is more amenable.